recipes-iceoryx2/iceoryx2/iceoryx2-validation-suite.inc

```diff
@@ -0,0 +1,86 @@
+SUMMARY = "iceoryx2"
+DESCRIPTION = "This package is used to build the iceoryx2 benchmarks, examples and tests"
+HOMEPAGE = "https://iceoryx.io"
+BUGTRACKER = "https://github.com/eclipse-iceoryx/iceoryx2/issues"
+LICENSE = "Apache-2.0 | MIT"
+LIC_FILES_CHKSUM = "file://LICENSE-APACHE;md5=22a53954e4e0ec258dfce4391e905dac \
+                    file://LICENSE-MIT;md5=b377b220f43d747efdec40d69fcaa69d"
+
+# Enable network for the compile task allowing cargo to download dependencies
+do_compile[network] = "1"
+
+DEPENDS = ""
+
+SRC_URI = "git://github.com/eclipse-iceoryx/iceoryx2.git;protocol=https;branch=main \
+           file://run-ptest"
+
+S = "${WORKDIR}/git"
+
+inherit cargo_bin ptest
+
+INSANE_SKIP:${PN} += "already-stripped"
+FILES_SOLIBSDEV = ""
+
+# This should actually be handled by meta-rust-bin most likely
+# in classes/cargo_bin.bbclass similar to Yocto Poky
+# meta/classes-recipe/rust-common.bbclass populating RUSTFLAGS
+# to prevent bitbake warnings "contains reference to TMPDIR [buildpaths]"
+# At this time, meta-rust-bin allows to pass in additional flags
+# via EXTRA_RUSTFLAGS to populate RUSTFLAGS
+RUST_DEBUG_REMAP = "--remap-path-prefix=${WORKDIR}=${TARGET_DBGSRC_DIR}"
+# See https://github.com/rust-embedded/meta-rust-bin/blob/master/classes/cargo_bin.bbclass
+# for variables to control the compilations
+EXTRA_RUSTFLAGS = "${RUST_DEBUG_REMAP}"
+CARGO_FEATURES = "libc_platform"
```

Comment:
While you are at it, using always the `iceoryx2/` prefix makes it consistent across all recipes and reduces the chance of a C&P error

```suggestion
CARGO_FEATURES = "iceroyx2/libc_platform"
```